### ISSUE TITLE
Frostbite Effect won't wear off.

### ISSUE THREAD

**Issue by FireBeastxD** [Electroblob77/Wizardry#656]
Minecraft version: 1.12.2
Wizardry version: 4.3.4
Environment: LAN Game

Issue details: When I'm casting Ice Age spell near my Allied Player, it will make them have Frostbite effect forever, so basically the Frostbite effect won't wear off and stuck at 00:00 until they kill themself. Tried jumping to see if it just a Icon bug but it wasn't (He can't jump) and also I noticed that the visual effect weren't showing from the start when I cast the Ice Age spell.

Other mods involved: I feel like nothing is involve with other mod but I have 50+ mod

**Comment by Electroblob77** [comment]
Interesting, normally this is just a client-side thing. Does re-logging help?

**Comment by FireBeastxD** [comment]
No. re-logging still doesn't help, all we can do is literally get killed. We're no longer playing anymore but what we did before was to remove the Spell that give Frostbite Effect and we just continue playing.
Here's the mod list that we play: https://pastebin.ubuntu.com/p/hV23zfyW5C/

**Comment by suneyedset** [comment]
I am playing single player on vech's super hostile modpack (spellbound caves 2), and I am having the same problem, over and over again. I summon a skeletal mage who then accidentally gets me with some cold spell, and then I have frostbite, and it won't go away unless I die. Tried teleporting, sleeping, remove curse spell, warm up gel packs. Nothing, only dying works.
